src/selenium/roomPage.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const RoomPage = Object.create(BasePage);
+
+RoomPage.checkIsolatedBookmark = function() {
+  const self = this;
+  const bookmarkSessionsIdsArr = ['3014'];
+  const visibleCheckSessionsIdsArr = ['3014', '3015', '2918'];
+
+  return self.bookmarkCheck(bookmarkSessionsIdsArr, visibleCheckSessionsIdsArr);
+};
+
+RoomPage.toggleSessionElem = function() {
+  const self = this;
+
+  // Checking the toggle behaviour of session having id 3014
+  const promise = new Promise(function(resolve) {
+    self.find(By.id('title-3014')).then(self.click).then(self.driver.sleep(1000)).then(function() {
+      const promiseArr = [];
+
+      promiseArr.push(self.find(By.id('desc-3014')).isDisplayed());
+      promiseArr.push(self.find(By.id('desc2-3014')).isDisplayed());
+      resolve(Promise.all(promiseArr));
+    });
+  });
+
+  return promise;
+};
+
+RoomPage.searchThenStarredMode = function() {
+  const self = this;
+  const idArr = ['3014', '2861', '3015'];
+  const promiseArr = idArr.map(function(elem) {
+    return self.find(By.id(elem));
+  });
+
+  return self.search('wel').then(self.toggleStarredButton.bind(self)).then(self.getElemsDisplayStatus.bind(null, promiseArr));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)